core/src/components/textarea/textarea.scss

```diff
@@ -103,6 +123,8 @@
   resize: none;
   appearance: none;
 
+  overflow: hidden;
```

Comment:
Does this fix the scrollbar showing when zooming in/out?

Reply:
This is to resolve the vertical scrollbar showing on Firefox. It may resolve other issues with the scrollbar as well though.